pyproject.toml

```diff
@@ -33,6 +33,7 @@ pyDes = "^2.0.1"
 PyJWT = "^2.5.0"
 pytz = ">=2020.1"
 redis = "^4.5.1"
+toml = "^0.10.2"
```

Comment:
Remember that for `0.x` versions, `^` only means up to `0.x+1`. I might here say: `">0.10.2,<1"`. It's true this could contain incompatible changes, as those are allowed in version 0, but really I think they creep in sometimes anyway, and a lot of times we might not be seeing a non-incompatible upgrade we want.

Reply:
Got it.